frontend/packages/noobaa-storage-plugin/src/components/activity-card/data-resiliency-activity/data-resiliency-activity.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { pluralize, Progress, ProgressSize } from '@patternfly/react-core';
+import { PrometheusResponse } from '@console/internal/components/graphs';
+import { getGaugeValue } from '../../../utils';
+import './data-resiliency-activity.scss';
+
+const getResiliencyProgress = (response: PrometheusResponse): number => {
+  const progress = getGaugeValue(response);
+  return Number((progress * 100).toFixed(1));
+};
+
+const getResiliencyEta = (response: PrometheusResponse): string => {
+  const eta = getGaugeValue(response);
```

Comment:
can we use https://github.com/openshift/console/blob/master/frontend/public/components/utils/datetime.ts#L93 and follow the same time convention used all over the console?